client/layout/masterbar/oauth-client.jsx

```diff
@@ -10,19 +10,19 @@ import PropTypes from 'prop-types';
  */
 import { addLocaleToWpcomUrl, getLocaleSlug } from 'lib/i18n-utils';
 
-const OauthClientMasterbar = ( { oauth2Client } ) => (
+const OauthClientMasterbar = ( { oauth2Client } ) => (
```

Comment:
Really minor: What happened here...?

Reply:
It was previously a non-breaking space. `}` requires the Shift key to type, so the following space was probably typed before Shift was released. Very easy to do. :-)